docs/server-admin.md

```diff
@@ -0,0 +1,37 @@
+# The administration panel
+
+Iodide has an administration view under `/admin`, based on the [Django
+admin](https://docs.djangoproject.com/en/2.1/ref/contrib/admin/) interface.
+
+To log into this interface, a user must either have the `is_staff` or `is_superuser` permission set. Most operations (including the one currently
```

Comment:
This wasn't true for me. I'm not allowed to view the */admin* page when only `is_superuser` is set.

Here's a complete breakdown of what I'm seeing: 

|State|Result|
|---|---|
|Only `is_superuser` set to true|Error when trying to access the */admin* page|
|Only `is_staff` set to true|The */admin* page loads and shows only a *Notebooks* link|
|Both set to true|The */admin* page loads with the following links: *Groups*, *Notebooks*, *Associations*, *Nonces*, *User social auths*|

Reply:
Er... I was on the wrong branch. 🙄